tests/test_multiplatform.py

```diff
@@ -475,6 +475,7 @@ def test_build_multiple_builds(
             cache_from=None,
             cache_to=None,
             pull=False,
+            target=None,
```

Comment:
These references should not be needed if it's already the default, right?

Reply:
That's what I thought, but the test failes if it's not there.  I guess the test is explicitly checking the parameter count that's provided too or something.

Reply:
Oh, I should have expanded this, I thought this was the method call to the func from the test method, but it's a mocked method, so yeah, the value would be set to none.